components/logins/src/error.rs

```diff
@@ -93,6 +96,9 @@ pub enum Error {
     #[error("Error executing SQL: {0}")]
     SqlError(#[from] rusqlite::Error),
 
+    #[error("SQL Connection Error")]
+    SqlConnectionError,
+
```

Comment:
Nit: I think this doesn't need to be added.

Reply:
cood catch! Removed the leftover.